dist/jdataview.js

```diff
@@ -1,23 +1,23 @@
-(function (factory) {
-	var global = this;
-
-	if (NODE || typeof exports === 'object') {
-		module.exports = factory(global);
-	} else
-	if (BROWSER) {
-		if (typeof define === 'function' && define.amd) {
-			define([], function () {
-				return factory(global);
-			});
-		}
-		else {
-			global.jDataView = factory(global);
-		}
-	}
-}(function (global) {
-
-'use strict';
-
+(function (factory) {
```

Comment:
That's weird, Github wouldn't show difference if it wouldn't be there. Maybe different newlines and local `git` ignores them? If you don't find the reason, I can try on my own a bit later.